sig-contributor-experience/community-maintenance.md

```diff
@@ -1,45 +1,53 @@
 # Community Maintenance 
 
-This document outlines maintenance tasks that SIG Contributor Experience should be doing on a regular basis.
-Copy this list into a new Github issue, for example "Community Maintenance Tasks for 2Q 2018". 
-Then people can volunteer to audit different parts of the project at once. 
-These tasks should be done at a _minimum of twice a year_, but ideally once a quarter.
+This document outlines maintenance tasks that SIG Contributor Experience should
+be doing on a regular basis. Copy this list into a new Github issue, for example
+"Community Maintenance Tasks for 2Q 2018".  Then people can volunteer to audit
+different parts of the project at once.  These tasks should be done at a
+_minimum of twice a year_, but ideally once a quarter.
 
-If any of these tasks can be automated, then do so, however most of these require a human to make a judgement decision. 
-If you find that any one person is in charge of a resource it is your responsibility to bring this issue to SIG Contributor Experience so that important parts of Kubernetes infrastructure are not assigned to one sole person. 
+If any of these tasks can be automated, then do so, however most of these
+require a human to make a judgement decision.  If you find that any one person
+is in charge of a resource it is your responsibility to bring this issue to SIG
+Contributor Experience so that important parts of Kubernetes infrastructure are
+not assigned to one sole person. 
 
-### Slack
+## Slack
 
 - [ ] Channels - Anything out of place or not following the guidelines? 
-  - [ ] Close unused channels. 
-  - [ ] Check to see if there is a purpose, pinned documents like agendas, and other best practices being used in the channel. These are not required but are useful.
+  - [ ] For community group channels - Check to see if there is a purpose,
+        pinned documents like agendas, and other best practices being used in
+        the channel. These are not required but are useful.
 - [ ] Audit the Integrations and apps
 - [ ] Audit the user tokens
 - [ ] Audit the Emojis
```

Comment:
So these three items have been automated already (thanks @Katharine).  Is there value in having them on this checklist?  I almost feel like we need a separate list of "things that are automated and we would be sad if they went away, but we don't need to actively pay attention to"

Reply:
We can probably remove the emoji and integrations checks, the user-tokens one we should still keep an eye on and ping folks about them -- like @mariantalla was using one to run a report but that's not likely needed to remain permanently.